BlazorApp/Pages/RegisterNewUser.razor

```diff
@@ -5,57 +5,118 @@
 @using Shared_Layer.ApiServices
 @inject HttpClient Http
 @inject IUserServices ApiService;
+@inject NavigationManager Navigation;
 
-<h2>Register User</h2>
-
-<EditForm Model="@model" OnValidSubmit="HandleRegistration">
-    <DataAnnotationsValidator />
-    <ValidationSummary />
-
-    <InputSelect @bind-Value="model.Role" class="form-control mb-3">
-        <option value="" disabled selected>Role</option>
-        <option value="Teacher">Teacher</option>
-        <option value="Student">Student</option>
-    </InputSelect>
-    <InputText @bind-Value="model.FirstName" placeholder="First Name" class="form-control mb-3" />
-    <InputText @bind-Value="model.LastName" placeholder="Last Name" class="form-control mb-3" />
-    <InputText @bind-Value="model.Email" placeholder="Email" class="form-control mb-3" />
-    <InputText @bind-Value="model.Password" placeholder="Password" class="form-control mb-3" />
-    <InputText @bind-Value="model.ConfirmPassword" placeholder="Confirm Password" class="form-control mb-3" />
-    <button class="btn btn-primary mt-3">Submit</button>
-
-    @if (!string.IsNullOrWhiteSpace(errorMessage))
-    {
-        <div class="alert alert-danger">
-            <strong>Error:</strong> @errorMessage
+<h2 class="text-center mt-5">Registration of user</h2>
+<h3 class="text-center mt-5"> fill in all fields to register user.</h3>
+
+<div class="container mt-4">
+    <!-- Smaller width to the form wrapper -->
+    <div class="row justify-content-center">
+        <div class="col-md-6">
+            <!-- Adjust column width -->
+            <EditForm Model="@model" OnValidSubmit="HandleRegistration">
+                <DataAnnotationsValidator />
+
+                <!-- Create a row for the form fields -->
+                <div class="row">
+
+                    <!-- Vänster Column -->
+                    <div class="col-md-6 d-flex flex-column justify-content-center">
+                        <div class="form-group">
+                            <label for="role">Role</label>
+                            <InputSelect id="role" @bind-Value="model.Role" class="form-control">
+                                <option value="" disabled selected>Select Role</option>
+                                <option value="Teacher">Teacher</option>
+                                <option value="Student">Student</option>
+                            </InputSelect>
+                        </div>
+
+                        <div class="form-group">
```

Comment:
<div class="form-group"> is old class and is not in use anymore, Chat gpt is often making this mistacke. Instead use  mb-3  for setting a bottom margin. or mt-3 for setting a top margin.
in Bootstrap 5, the form-group class has been removed in favor of more utility classes and simplified form layouts. The same layout and spacing can now be achieved using margin utility classes, such as mb-3 (which means "margin-bottom: 3"), along with form-control for inputs.

Reply:
ändrade detta på alla inputfields